syntax.d/zsh.ini

```
@@ -0,0 +1,7 @@
+name=ZSH
+extensions=zsh
+highlight_numbers=true
+singleline_string_quotes='
+singleline_comment_start=\#
```

I think this shouldn't be escaped
```suggestion
singleline_comment_start=#
```